docs/project/2024/alkhatib.hoda/index.md

```diff
@@ -0,0 +1,112 @@
+
+# Snake game project 
+The game 'Snake', displayed on an 8×8 LED Matrix module 
+
+:::info
+
+**Author**: Alkhatib Hoda \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-AlkhatibHoda
+
+:::
+
+## Description
+
+My project is a recreation of the popular game 'Snake,' displayed on an 8×8 LED Matrix module using a Raspberry Pi.My aim is to create something that appears simple but holds many memories of my childhood and provides enjoyable entertainment.
+
+## Motivation
+
+Creating a Snake game with Rust on a Raspberry Pi and LED matrix blends nostalgia with technical challenge, fostering both creativity and learning.
+
+## Architecture
+
+![Architecture.jpg](https://github.com/AlkhatibHoda/upb-fils-ma.github.io/assets/163418706/9adce84e-7836-41bc-a30b-dc0bfab75834)
```

Comment:
Add "Architecture.jpg" to the comitted files and update the link accordingly (e.g (./Architecture.jpg))

Reply:
Use the local file.